File: smallest-index-with-equal-value/smallest-index-with-equal-value.py
class Solution:
    def smallestEqual(self, nums: List[int]) -> int:
        res = []
        for i in range(len(nums)):
            if i % 10 == nums[i] :
                res.append(i)
        if len(res)==0:
            return -1
        else:
            return min(res)

File: smallest-index-with-equal-value/smallest-index-with-equal-value.cpp
class Solution {
public:
    int smallestEqual(vector<int>& nums) {
        vector<int>res;
        for(int i = 0 ; i < nums.size() ; i++){
            if(nums[i]==i%10){
                res.push_back(i);
            }
        }
        if(res.size()==0)
        return -1 ;
        else
        return *min_element(res.begin(),res.end());
    }
};